organization/ircam_oauth2_provider/views.py

```diff
@@ -0,0 +1,151 @@
+import requests
+from allauth.socialaccount.models import (SocialAccount,SocialLogin)
+from allauth.socialaccount.providers.oauth2.views import (OAuth2Adapter, OAuth2LoginView, OAuth2CallbackView)
+from .provider import IrcamAuthProvider
+from django.conf import settings
+from django.contrib.auth.models import User
+from organization.network.models import Person
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import IntegrityError
+
+import logging
+
+class Logger:
+
+    def __init__(self,level):
+        self.logger = logging.getLogger('IrcamOauth2')
+#        self.hdlr = logging.FileHandler(file)
+        self.hdlr = logging.StreamHandler()
+        self.formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+        self.hdlr.setFormatter(self.formatter)
+        self.logger.addHandler(self.hdlr)
+        self.logger.setLevel(level)
+
+    def infoP(self, prefix, message):
+        self.logger.info(' ' + prefix + ' : ' + message)
+    def warningP(self, prefix, message):
+        self.logger.warning(' ' + prefix + ' : ' + message)
+    def errorP(self, prefix, message):
+        self.logger.error(prefix + ' : ' + message)
+    def debugP(self, prefix, message):
+        self.logger.debug(prefix + ' : ' + message)
+    def info(self, message):
+        self.infoP('IrcamAuth',message)
+    def warning(self, message):
+        self.warningP('IrcamAuth',message)
+    def error(self, message):
+        self.errorP('IrcamAuth',message)
+    def debug(self, message):
+        self.debugP('IrcamAuth',message)
```

Comment:
Why defining a logger here instead of using the generic/app logger ?

Reply:
done